src/Library.fs

```diff
@@ -47,7 +47,7 @@ type ChangelogExtensions =
     static member Unwrapped(sections: ChangelogSectionCollection) =
         sections
         |> Seq.choose (fun section ->
-            match SemVersion.TryParse section.MarkdownVersion with
+            match SemVersion.TryParse(section.MarkdownVersion, SemVersionStyles.Any) with
```

Comment:
what styles were failing before this change?

Reply:
None, but `SemVersion.TryParse` without styles provided is deprecated. I couldn't really think of a reason to be stricter. The strictness would prohibit whitespace, prohibit `v` prefixes, and prohibit omiting minor/patch numbers.